Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:latest
+LABEL maintainer="seglh"
```

Comment:
has the docker image been pushed to the seglh dockerhub?

Reply:
Not yet. If you are happy with the script, I can push it to seglh dockerhub.

Reply:
great, thanks